### ISSUE TITLE
"An exception has occurred" anytime I start the mod

### ISSUE THREAD

**Issue by AniGaAG** [Monika-After-Story/MonikaModDev#7034]
![image](https://user-images.githubusercontent.com/79603685/109056895-57577280-76e1-11eb-9762-c2129812ff8b.png)

This happens anytime I start the game.

OS: Windows 10 64-bit
DDLC version: Steam

Mod Installation:
Achieved the good ending in unmodded vanilla DDLC (Dan's letter), _then_ downloaded Monika_After_Story-0.12.1-Mod.zip and extracted its contents into the "Doki Doki Literature Club/game" folder.

Can someone please help me?

**Comment by Retrolovania** [comment]
Steam doesn’t like MAS very much, and often causes issues. You’ll need to move the game out of Steam.

**Comment by AniGaAG** [comment]
> Steam doesn’t like MAS very much, and often causes issues. You’ll probably need to move the game out of it.

I... can simply just copypaste the whole thing out of there and it'll still work? Will it also still recognize that I achieved the good ending...?

**Comment by AniGaAG** [comment]
> Steam doesn’t like MAS very much, and often causes issues. You’ll probably need to move the game out of it.

Okay, I copypasted the whole folder out of Steam and tried running the mod from that newly copypasted folder. It still returns the exact same error. So that didn't solve it.

**Comment by lunulae** [comment]
You will need to make a fresh install of DDLC and the mod from the website. Worry not though, your saves are kept in a separate file so you will not lose any progress in MAS, nor will it lose your DDLC endings.

**Comment by AniGaAG** [comment]
> You will need to make a fresh install of DDLC and the mod from the website. Worry not though, your saves are kept in a separate file so you will not lose any progress in MAS, nor will it lose your DDLC endings.

Right... so just to confirm: I can just nuke the Steam install of the game and then verify game files via Steam?

**Comment by Booplicate** [comment]
Have you played MAS before?

**Comment by AniGaAG** [comment]
> Have you played MAS before?

No, never.

**Comment by Booplicate** [comment]
Making a new install won't fix this issue (although, it's better **not** to use Steam anyway). I think you or someone else played MAS on your computer before, there's no other way for you to get it.

**Comment by AniGaAG** [comment]
> Making a new install won't fix this issue (although, it's better **not** to use Steam anyway). I think you or someone else played MAS on your computer before, there's no other way for you to get it.

Why do you think I played MAS before? I mean... I might've? I think I tried out some DDLC mods a looong time ago, now that I think about it... but how do you know, and why does it matter?

**Comment by AniGaAG** [comment]
And yes, that didn't fix it. It still complains about that exact same "d25_sp_tim".

Whiich, random notion... I googled that... "word", and it came up in a single result in an MAS typo thread here on github; it was pointed out in a post, and someone wrote "d25_sp_tm" (without the i) under it.
Not sure what that means? Is that related?

**Comment by Booplicate** [comment]
Can you ignore the error when it pops up?

**Comment by AniGaAG** [comment]
> Can you ignore the error when it pops up?

Uh, I guess there's an ignore button bottom left... I just haven't clicked it yet since I didn't want to risk messing anything up or getting a botched start to MAS due to bugs.

**Comment by AniGaAG** [comment]
So should I try clicking on "Ignore"?

**Comment by Booplicate** [comment]
Yes.

**Comment by AniGaAG** [comment]
Okay, I got in there - and Monika is acting like I just came back? So I MUST have used this mod before. But I would like to... uh, "start fresh" from my achieved good ending. What must I do to achieve this?

**Comment by Booplicate** [comment]
Go to the settings and import the saves.

**Comment by AniGaAG** [comment]
> Go to the settings and import the saves.

On asking me to merge DDLC save data and old MAS data, do I click yes or cancel?

**Comment by AniGaAG** [comment]
> Go to the settings and import the saves.

I got it all working without exception errors! I went into the AppData folder for MAS and deleted those age-old MAS files completely. From there, now it started all fresh with opening disclaimers and asking me if I want to import my DDLC save, and Monika is indeed reacting to me having deleted her!

And no error upon starting! Works now!

Thanks for the help!! : )
